styles/main.scss

```diff
@@ -119,6 +119,208 @@ $font-sizes: (
   }
 }
 
+// Purchase page classes
+#purchase {
+  .book-description {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    background-color: #6A92C1;
+    text-align: center;
+    padding-bottom: 100px;
+    padding-left: 25px;
+    padding-right: 25px;
+    align-items: center;
+    @media screen and (max-width: 700px) {
+      justify-content: center;
+      text-align: center;
+    }
+
+    .book-image {
+      max-width: 550px;
+      .book-front-cover {
+        flex-grow: 1;
+        max-width: 100%;
+        max-height: 100%;
+        padding-top: 60px;
+      }
+    }
+    .book-text {
+      padding-top: 200px;
+      padding-left: 100px;
+      h3 {
+        display: none;
+      }
+
+      .purchase-links {
+        button {
+          background-color: #07365B;
+          color: #FFFFFF;
+          height: 40px;
+          width: 308px;
+          border-radius: 4px;
+          border: unset;
+          font-family: Proxima Nova;
```

Comment:
Just had a chat with the the branding team regarding fonts - the desire is for Proxima Nova and Adelle,
so I was thinking we would override the bootstrap font-family and use one as the default, and the other would be configured in main.scss.
In any case, since the two fonts are too expensive to buy and outright include in the project, we will be getting a font stack from the branding team to use for both Proxima Nova and Adelle, but that will probably come later?

Reply:
Yeah, these are merely placeholders.  I can do a complete search and replace once we have the official font.